style.css

```diff
@@ -127,3 +127,86 @@ button:hover {
 .button-2 {
   margin-top: 25px;
 }
+
+.contact-main {
+  width: 100%;
+  border-top-left-radius: 120px;
+  background-color: #6070ff;
+  background-image: url(./images/contact-shape.svg);
+  background-repeat: no-repeat;
+  background-size: 70% 100%;
+  background-position-x: right;
+}
+
+.contact-me {
+  width: 92%;
+  height: auto;
+  display: flex;
+  flex-direction: column;
+  padding-top: 160px;
+  margin: auto;
+}
```

Comment:
- [ ] Great job 👍. It's better to layout this part of the form using `flexbox`, as it is one of the requirements.

Reply:
@Wes-Isaac Hi Issac, Thank you so much for your valuable feedback and for guiding me. I need some more clarification on this. I already used flexbox here(line no 144). What exactly do you want me to do now, can you give me an example?